tests/Data/Stubs/Database/User.php

```diff
@@ -31,4 +37,9 @@ public function groups(): BelongsToMany
     {
         return $this->belongsToMany(Group::class);
     }
+
+    public function searchableUsing(): Engine
+    {
+        return class_exists('\Laravel\Scout\Engines\DatabaseEngine') ? new DatabaseEngine() : new NullEngine();
```

Comment:
If this would only be supported with DatabaseEngine, it think we should throw an error with a clear explanation why this doesn't work on older scout versions

Reply:
DatabaseEngine is the only engine we can use to run tests. 

In real usage we doesnt care what engine is used by Scout, as it incapsulates behind facade.

Reply:
Older Scout versions provides only NullEngine (that is searches nothing) and AlgoliaEngine (that is not applicable for testing, but may be used in real applications)

Reply:
okay thanks